squad-server/log-parser/player-disconnected.js

```diff
@@ -0,0 +1,19 @@
+export default {
+    regex: /^\[([0-9.:-]+)]\[([ 0-9]*)]LogEasyAntiCheatServer: \[[0-9:]+]\[[A-z]+]\[EAC Server] \[Info]\[UnregisterClient] Client: ([A-z0-9]+) PlayerGUID: ([0-9]{17})/,
+    onMatch: (args, logParser) => {
+        const playerTrackerObjectName = `player-name-tracker-${args[4]}`;
+
+        const data = {
+            raw: args[0],
+            time: args[1],
+            chainID: args[2],
+            playerSuffix: logParser.eventStore[playerTrackerObjectName] ? logParser.eventStore[playerTrackerObjectName] : 'Unknown',
```

Comment:
There's probably no need to be working with the player's suffix here (or anywhere in this PR).

The player's suffix is collected using two log events and then is added into their player object listed in the `SquadServer.players` array. As this log line has their SteamID in the `SquadServer` class, where you forward the event, you can use the `.getPlayerBySteamID()` method to get the player's object. Then you can probably remove all other references to the player's suffix from this PR.

Reply:
Oh, okay, I did not know it. I will remove that part.

Reply:
Alright, done. I've removed the reference to the player suffix.
And I am already using `.getPlayerBySteamID()` in `squad-server/index.js`.